tests/libtest/env.py

```diff
@@ -267,29 +267,29 @@ def enable_auth(workspace):
     an auth-enabled server.
 
     Running the tests only work if the initial value (in package
-    session_config.json) is FALSE for authentication.enabled.
+    server_config.json) is FALSE for authentication.enabled.
     """
 
-    session_config_filename = "session_config.json"
+    server_config_filename = "server_config.json"
 
     cc_package = codechecker_package()
     original_auth_cfg = os.path.join(cc_package,
                                      'config',
-                                     session_config_filename)
+                                     server_config_filename)
 
     shutil.copy(original_auth_cfg, workspace)
 
-    session_cfg_file = os.path.join(workspace,
-                                    session_config_filename)
+    server_cfg_file = os.path.join(workspace,
+                                   server_config_filename)
 
-    with open(session_cfg_file, 'r+') as scfg:
+    with open(server_cfg_file, 'r+') as scfg:
         __scfg_original = scfg.read()
```

Comment:
Can't we use `load_json_or_empty` here? Do we need the seek and `__scfg_original`?

Reply:
Also, do we use `scfg` for anything at all below?

Reply:
Doesn't seem like it. This is a copy from the real session manager file, in which case it was used.